NEWS.md

```diff
@@ -7,7 +7,9 @@
   unique time points were available for any subject. Now it is checked that there are
   no duplicate time points per subject, and this is explained also in the
   function documentation and the introduction vignette.
-
+- Previously in `mmrm` calls, the `weights` object in the environment where formula is
+  defined will be replaced by the weight used. Now this behaviour is removed and your variable
```

Comment:
is `behaviour` misspelled? I saw spell check is complaining...

Reply:
if we're using American English spellings, it'd be behavior. Otherwise, behaviour is fine (British English). Typically we'd just strive for consistency across the documentation. @danielinteractive, have we been having this is mind?

Reply:
spelling updated

Reply:
I think we should use American English. Not sure if we can define that for the spellcheck @cicdguy any tips?

Reply:
perfect thx @cicdguy

Reply:
hm but we have that setting already. not sure why it was not caught by spellcheck then